_includes/articles/river-descriptors-psbt.md

```diff
@@ -0,0 +1,56 @@
+{:.post-meta}
+*by [Philip Glazman][], Engineer at [River Financial][]*
+
+*River Financial​ is a challenger financial institution specializing in Bitcoin
+financial services. River Financial’s flagship product, a Bitcoin brokerage,
+provides retail investors with a high-touch platform to buy and sell Bitcoin.*
+
+River Financial leverages two technologies in its wallet software: [Partially
+Signed Bitcoin Transactions (PSBTs)][topic psbt] and [Output Script
+Descriptors][topic descriptors]. The decision to incorporate these standards has
+introduced greater flexibility and interoperability in River’s wallet operations.
+
+The decision to bring PSBTs into the wallet stack was influenced by the concept
+of treating key signers as interfaces. PSBT is a format for signers described in
+[BIP174][] authored by Andrew Chow. Before this standard, there had been no
+standardized format to describe an unsigned transaction. As a consequence, each
+signer used vendor-specific formats which could not interoperate. By conforming
+to the PSBT standard, the wallet infrastructure can avoid vendor lock-in, reduce
+the attack surface in the signer logic, and have better guarantees about the
+transaction being created by the wallet. The standard has also made
+multisignature scripts safer to use, therefore significantly improving security
+without a notable increase in operational expense.
+
+The decision to implement Output Script Descriptors in the wallet software has
+greatly reduced the complexity in wallet operations and has improved flexibility
+in the feature set. Descriptors is a language for describing scripts that was
```

Comment:
"flexibility" is already stated in the second and in the last paragraph and it's somewhat unclear what "flexibility in the feature set" means here before reading the bullet points below...perhaps ease/speed of new/custom feature development, expanded the possible feature set, or allowed a wider range of features.